src/extensions/default/Microsoft.DotNet.UpgradeAssistant.Steps.Source/CodeFixerStep.cs

```diff
@@ -129,7 +135,9 @@ protected override async Task<UpgradeStepApplyResult> ApplyImplAsync(IUpgradeCon
                 var newDiagnosticCount = Diagnostics.Count();
                 if (diagnosticCount == newDiagnosticCount)
                 {
-                    Logger.LogWarning("Diagnostic {DiagnosticId} was not fixed as expected. This may be caused by the project being in a bad state (did NuGet packages restore correctly?) or by errors in analyzers or code fix providers related to this diagnostic.", DiagnosticId);
+                    var description = $"Diagnostic {DiagnosticId} was not fixed as expected. This may be caused by the project being in a bad state (did NuGet packages restore correctly?) or by errors in analyzers or code fix providers related to this diagnostic.";
+                    Logger.LogWarning(description);
+                    AddResultToContext(context, DiagnosticId, string.Empty, "Success", description);
```

Comment:
based on the description, this doesn't seem like Success...

Reply:
Good catch. My bad.